src/Tippy.js

```diff
@@ -37,14 +42,48 @@ class Tippy extends React.Component {
     }
   }
 
+  get isManualTrigger() {
+    return this.props.trigger === 'manual'
+  }
+
   componentDidMount() {
     this.setState({ isMounted: true })
+
     this.tip = tippy.one(ReactDOM.findDOMNode(this), this.options)
-    this.props.onCreate && this.props.onCreate(this.tip)
+
+    const { onCreate, isEnabled, isVisible } = this.props
+
+    if (onCreate) {
+      onCreate(this.tip)
+    }
+
+    if (isEnabled === false) {
+      this.tip.disable()
+    }
+
+    if (this.isManualTrigger && isVisible === true) {
+      this.tip.show()
```

Comment:
Per atomiks/tippyjs#230, should this be?
```suggestion
      setTimeout(this.tip.show)
```

Reply:
I'm going to release a new `tippy.js` patch that changes the global click listener to use capture phase (instead of bubbling) by default. That will prevent the need to do this.

Reply:
May want to update the docs on how `isVisible` and `showOnInit` relate (or don't) to help avoid confusion down the road. Nice job on the project though and thank you for all the work.